getting-started/installation-windows.rst

```diff
@@ -66,15 +66,22 @@ Install ``gcc`` for 64-bit environment
 
     :: Proceed with installation? [Y/n] y
     
-    
+Install ``gcc`` 
+
+
+::
+
+    $ pacman -S gcc
+        
 Check everything is installed properly,
 
 ::
 
-    $ pacman -Q make perl mingw-w64-x86_64-gcc
+    $ pacman -Q make perl mingw-w64-x86_64-gcc gcc
     make 4.2.1-1
     perl 5.30.0-1
     mingw-w64-x86_64-gcc 9.2.0-2
+    gcc 9.1.0-2
```

Comment:
What's the difference between these two copies of gcc? It surprises me slightly that they have two different version numbers, which one is actually being used?

Reply:
If I only install gcc and not mingw-gcc, make fails giving lots of error.
If I only install mingw-gcc and not gcc, make fails whenever "gcc" or "g++" commands are used.

I am also confused. But keeping both version only works. I failing to attach and error output file, I can send you via email if you want to see it.

Reply:
`gcc` is the native MSYS2 compiler, `mingw-w64-x86_64-gcc` is the 64bit native MinGW compiler tool chain.

While MSYS2 is a subsystem based on cygwin and focuses on providing a Unix-style environment and package manager for Windows, the MinGW tools provide and create Windows programs that are independent from the subsystem used.

For EPICS compilation we want to use the MinGW tool chains (I have 32bit and 64bit working). I am not sure if and what for the MSYS2 gcc is really necessary. Note that the built-in EPICS platform detection script is failing to detect the MinGW toolchains correctly. EPICS_HOST_ARCH has to be set manually. (That should be fixed in base, probably.)

https://www.msys2.org/wiki/MSYS2-introduction/
has a good overview of the MSYS2 / MinGW ecosystem. (Might be useful to link to.)

Reply:
See https://bugs.launchpad.net/epics-base/+bug/1873406 for the EPICS_HOST_ARCH detection.